### DISCUSSION TITLE
webui not accessible after Valetudo 2024.10.0

### DISCUSSION THREAD

**Discussion by googol42** [Hypfer/Valetudo#2115] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, for a long time

### Your questions

I just updated to 2024.10.0. I think that my update got corrupted: Normally I perform any Valetudo updates in the browser, this time I executed it on my mobile phone and accidentally reloaded the page while the update was in progress.

I still can SSH into my robot (dreame z10 pro). I can operate it via the hardware buttons, but I cannot use the web ui.

Can you help me getting my robot back to life?

PS: I promise not to perform any Valetudo updates on my phone anymore ;-)

**Comment by Hypfer** [comment, marked as answer]
Weird. The phone should not affect that at all, as the update happens inside the robot.

Oh well.
Essentially, you just need to follow the same steps as on the initial install.

Or just put the right binary in the right place:

1. Ensure that `/data/valetudo` is the expected binary in its uncorrupted form.
    Just download a new one there to be sure. Your Z10 will need the arm64 build. Don't forget to make it executable
2. Ensure that `/data/_root_postboot.sh` still exists and is also executable.
    You can find that script in `/misc/_root_postboot.sh.tpl` if it's missing

**Reply by googol42** [reply]
Thanks for you fast response. `/data/valetudo` was missing. Downloading, marking it as executable and rebooting did the trick. Thank you :+1:

**Reply by googol42** [reply]
> Weird. The phone should not affect that at all, as the update happens inside the robot.

After (accidentally) reloading the page, I received a message that the process was aborted. I then triggered the update again. Maybe that interfered with the (still running) first update?
